packages/core/admin/admin/src/content-manager/components/DynamicTable/index.js

```diff
@@ -1,15 +1,22 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
-import { DynamicTable as Table, useStrapiApp } from '@strapi/helper-plugin';
+import {
+  TableProvider,
+  TableEmptyBody,
+  TableActionBar,
+  TableHead,
+  TableHeaders,
```

Comment:
I think it would be better if we could avoid having too many top-level exports in the helper plugin for things that can only be used within a table.

In my opinion the best API would be to export a single Table component, similarly to what [our own Menu](https://design-system-git-main-strapijs.vercel.app/?path=/docs/design-system-components-v2-simplemenu--basic#menu) and [Combobox](https://github.com/strapi/design-system/tree/main/packages/primitives#basic-usage) components are doing.

It would look somewhat like this:

```js
import { Table } from "@strapi/helper-plugin"

return (
  <Table.Provider>
      <Table.ActionBar />
      <Table.Rows />
  </Table.Provider>
)
```

Reply:
I had something like that but @joshuaellis had [suggested the named exports](https://github.com/strapi/strapi/pull/16806#discussion_r1202035896). Maybe I misunderstood 🤔